content/docs/0.8.1/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -112,6 +113,13 @@ We provides two testing purpose backupstore based on NFS server and Minio S3 ser
 
 **Result:** Longhorn can store backups in S3. To create a backup, see [this section.](../create-a-backup)
 
+### Using a self signed SSL certificate for S3 communication
+If you want to use a self signed ssl certificate you can expose an `AWS_CERT` environment variable that contains your public certificate to the longhorn pods.
+We recommend you do so by adding the AWS_CERT key/value pair to the `minio-secret` or `aws-secret` and making it available to the `longhorn-system` namespace.
+For an example of using a self signed ssl certificate, have a look at our [minio-backupstore.yaml](https://github.com/longhorn/longhorn/blob/master/deploy/backupstores/minio-backupstore.yaml).
```

Comment:
`self-signed SSL`

`or see the example in ` (insert anchor link to `Set up a Local Testing Backupstore`.)

Reply:
fixed also made the NFS change.